crates/forge/bin/cmd/test/mod.rs

```diff
@@ -604,6 +612,47 @@ fn list(
     Ok(TestOutcome::new(BTreeMap::new(), false))
 }
 
+/**
+ * Creates table summary header
+ */
```

Comment:
Let's convert this to actual rust docstrings `///`, instead of using a JS format :D

Reply:
Fixed